src/test/compile-fail/lint-exceeding-bitshifts.rs

```diff
@@ -11,7 +11,7 @@
 #![deny(exceeding_bitshifts)]
 #![allow(unused_variables)]
 #![allow(dead_code)]
-#![feature(num_bits_bytes)]
+#![feature(num_bits_bytes, const_indexing)]
```

Comment:
Oh, interesting ... am I right in inferring that this program will compile *successfully* (and the error occurs at runtime, assuming the checks are turned on), if the feature gate is not included here?